simulation/src/LeiosProtocol/Short/Node.hs

```diff
@@ -964,9 +995,6 @@ mkBuffersView cfg st = BuffersView{..}
     let tryCertify eb = do
           Certified{cert} <- Map.lookup eb.id votesForEB
           guard (not $ Set.member (endorseBlockPipeline cfg.leios eb) pipelinesInChain)
-          -- Note: we expect to have received the IBs for any
```

Comment:
@SupernaviX When issuing a Ranking Block, does the Rust simulator skip over certs if it doesn't already have all of the IBs that the cert transitively includes?

Reply:
Resolving this Conversation since we're discussing this elsewhere (ie on PR 427).